src/Microsoft.ML.Onnx/TransformerChainOnnxConverter.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Collections.Generic;
+using Microsoft.ML.Core.Data;
+using Microsoft.ML.Data;
+using Microsoft.ML.UniversalModelFormat.Onnx;
+
+namespace Microsoft.ML.Model.Onnx
+{
+    public class TransformerChainOnnxConverter
```

Comment:
>public class [](start = 4, length = 12)

static class

Reply:
Done.

---
In reply to: [245213997](https://github.com/dotnet/machinelearning/pull/2013#discussion_r245213997) [](ancestors = 245213997)